### PR TITLE
Update instructions for "Fall back to AWS CloudFront"

### PR DESCRIPTION
**Pull request by matthillco**
This has been updated based on a 2ndline drill. The distinction between actions to take on **production** vs **staging** were unclear and have been improved. I've also added a couple of new details regarding how to do the manual failover, and how to check if the fallback domain is working. Bullet lists have been split up and nesting improved/removed for clarity.

### CODE REVIEW

File: source/manual/fall-back-to-aws-cloudfront.html.md [hunk 1]
```diff
@@ -23,49 +23,78 @@ This backup CDN is currently provided by AWS CloudFront.
...
+
+### Next steps
+
+In either case, the records should look like `d0000000000000.cloudfront.net.` (with 0s replaced with letters and numbers). Now you can manually update the `CNAME` records for both domains in both GCP and AWS, via the tabs you opened in your web browser earlier:
+
+- Change the canonical name from `www-gov-uk.map.fastly.net.` to the CloudFront domain name you found before, including the trailing period (e.g. `d0000000000000.cloudfront.net.`)
+- Test if the new Cloudfront domain is serving assets correctly, for example:
+
+```bash
+curl -vs https://assets.staging.publishing.service.gov.uk/media/662a74aa45f183ec818a72c2/dvsa-earned-recognition-vehicle-operators-accredited-list.csv/preview | grep cloudfront
```
Comment (kentsanggds) [inline source/manual/fall-back-to-aws-cloudfront.html.md:94]: I think we need to tell the dev what to look for from the curl response that will inform them that the requests are being served by the updated CDN - 

missing key - `fastly-backend-name`

or the presence of `xxxxx.cloudfront.net` value in the `via` key.

File: source/manual/fall-back-to-aws-cloudfront.html.md [hunk 2]
```diff
@@ -23,49 +23,78 @@ This backup CDN is currently provided by AWS CloudFront.
...
+```
+
+- Perform a manual failover, for example by temporarily changing the new canonical domain name with a random prefix.
```
Comment (kentsanggds) [inline source/manual/fall-back-to-aws-cloudfront.html.md:97]: As we are now checking where the requests are being served from in the curl response we probably don't need this step to break it.

File: source/manual/fall-back-to-aws-cloudfront.html.md [hunk 3]
```diff
@@ -23,49 +23,78 @@ This backup CDN is currently provided by AWS CloudFront.
...
 - **On production only**, after performing the manual failover, you should also update our infrastructure-as-code to match the changes you just made:
```
Comment (kentsanggds) [inline source/manual/fall-back-to-aws-cloudfront.html.md:98]: Should this section be in the `Production` section?

**Review by kentsanggds — CHANGES_REQUESTED** [review]
Just a few changes needed before it's good to go.

File: source/manual/fall-back-to-aws-cloudfront.html.md (hunk 3 above)
Comment (matthillco) [reply source/manual/fall-back-to-aws-cloudfront.html.md:98]: Do you mean move it to that section, or duplicate it? I'm unclear. @kentsanggds

Comment (matthillco) [reply source/manual/fall-back-to-aws-cloudfront.html.md:98]: Actually, I'm not sure we need to implement your comment: this point is in the "Next Steps" section which applies to both Staging and Production. @kentsanggds

File: source/manual/fall-back-to-aws-cloudfront.html.md (hunk 1 above)
Comment (matthillco) [reply source/manual/fall-back-to-aws-cloudfront.html.md:94]: Updated with your suggestions

File: source/manual/fall-back-to-aws-cloudfront.html.md (hunk 2 above)
Comment (matthillco) [reply source/manual/fall-back-to-aws-cloudfront.html.md]: Updated with your suggestion.

File: source/manual/fall-back-to-aws-cloudfront.html.md (hunk 3 above)
Comment (kentsanggds) [reply source/manual/fall-back-to-aws-cloudfront.html.md:100]: I think that section needs to be moved as it only applies to production. 
I don't think for Staging its necessary to update the terraform as part of the drill, unless we want to test the changes on Staging first before applying to Production if there is a real need to switch CDN?

**Review by kentsanggds — APPROVED** [review]

**Pull request merged**